nixnet/session.py

```diff
@@ -33,7 +62,32 @@ def __init__(
             list,
             interface,
             mode):
-        "http://zone.ni.com/reference/en-XX/help/372841N-01/nixnet/nxcreatesession/"
+        """Create an XNET session at run time using strings.
+
+        This function creates a session using the named database objects
+        specified in 'list' from the database named in 'database_name'.
+
+        Args:
+            database_name: The XNET database to use for interface configuration.
+                The database name must use the <alias> or <filepath> syntax
+                (refer to Databases).
+            cluster_name: The XNET cluster to use for interface configuration.
+                The name must specify a cluster from the database given in the
+                database_name parameter. If it is left blank, the cluster is
+                extracted from the list parameter; this is not allowed for modes
+                of :class:`nixnet._cconsts.NX_MODE_FRAME_IN_STREAM` or
+                :class:`nixnet._cconsts.NX_MODE_FRAME_OUT_STREAM`.
```

Comment:
Why are the constants and not enum values being references?

Those constants are an implementation detail and we should act like they don't exist in public-facing documentation.